incorrectbundle/3scale-community-operator/3scale-community-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+packageName: 3scale-community-operator
```

Comment:
Can we put this in a `testdata` directory under the package being tested (`sqlite`)? I don't think we want to put bad test manifests at the root of the repo.